downstream/modules/platform/proc-edge-manager-image-pullsecrets.adoc

```diff
@@ -0,0 +1,34 @@
+[id="edge-manager-image-pullsecrets"]
+
+= Optional: Using image pull secrets
+
+If your device relies on containers from a private repository, you must configure a pull secret for the registry. 
+Complete the following steps:
+
+.Procedure
+
+. Depending on the kind of container image you use, place the pull secret in one or both of the following system paths on the device:
++
+* Operating system images use the `/etc/ostree/auth.json` path.
+* Application container images use the `/root/.config/containers/auth.json` path.
++
+[IMPORTANT]
+=====
+The pull secret must exist on the device before the secret can be consumed.
+=====
+
+. Ensure that the pull secrets have the following format:
```

Comment:
Suggest changing to "...use the following format:" to make it clear that a specific structure is applied rather than possessed. 

Ensure that the pull secrets use the following format: